### ISSUE TITLE
Issue Selecting Many Albuns for Backup

### ISSUE THREAD

**Issue by JoseFilipeFerreira** [immich-app/immich#12738]
### The bug

When selecting a large amount of albums for backup (118) containing a large amount of assets (19839) and exiting the selection screen most of the albums are not selected.
If this process is repeated over and over again, less and less albums are skipped. Until all are selected.
However, after half of the backup is completed, all of the albums disappeared randomly.

### The OS that Immich Server is running on

Docker 27.2.0, build 3ab4256958 + Arch 6.10.5-arch1-1

### Version of Immich Server

v1.115.0

### Version of Immich Mobile App

1.115.0 build.159

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Can provide later if relevant
```

### Your .env content

```Shell
Can provide later if relevant
```

### Reproduction steps

1. have too many albuns and assets
2. attempt to select all albuns for backup
3. exit selection screen
4. see that most of the albuns are not selected

### Additional information

Even if I wait more time before exiting the selection screen there seems to be no change to the amount of selected albums.

**Comment by wilcochris** [comment]
I am experiencing the same issue. I have almost 200 albums on my phone and I was told this was to do with the amount of assets/albums I have and that they would be looking into it soon.

I was also told that they are selected but that the app needs to catch up (I don't believe this to be the case) for them to show as selected

**Comment by alextran1502** [comment]
Fixed in #12784
